idc/settings.py

```diff
@@ -565,3 +565,7 @@ def GET_BQ_COHORT_SETTINGS():
     ]
     SHOW_TOOLBAR_CALLBACK = True
     INTERNAL_IPS = (os.environ.get('INTERNAL_IP', ''),)
+#OHIF_SETTINGS
+
+APPEND_SLASH = False
+# default is to add trailing '/' to urls ie /callback becomes /callback/. Ohif does not like /callback/ !
```

Comment:
We'll need to keep an eye on this one, because (as you've probably noticed) nearly all of our routes require a slash.

Reply:
Understood. This was an annoying problem that took up several hours to diagnose. At least I learned more about oauth, react, and django. We could probably put the OHIF component in its own app.